src/colors.css

```
@@ -58,4 +58,69 @@ html[data-color-scheme="dark"] {
   color-scheme: dark;
 }
 
-/** TODO: Add colors for high contrast modes */
+html[data-color-scheme="light-high-contrast"], html:not([data-color-scheme]) {
+  --color-neutral05: #fff;
+  --color-neutral10: #fff;
+  --color-neutral15: #fff;
+  --color-neutral20: #000;
+  --color-neutral25: #000;
+  --color-neutral30: #000;
+  --color-neutral40: #000;
+  --color-neutral50: #000;
+  --color-neutral60: #000;
+  --color-neutral70: #000;
+  --color-neutral80: #000;
+  --color-neutral90: #000;
+
+  --color-danger0: #a5102d;
+  --color-danger1: #a5102d;
+  --color-danger2: #a5102d;
+  --color-danger4: #a5102d;
+  --color-danger5: #a5102d;
+
+  --color-accent8: #000099;
+  --color-accent7: #000099;
+  --color-accent6: #000099;
+  --color-accent5: #000099;
+  --color-accent4: #000099;
+
+  --color-focus: #000099;
+  color-scheme: light-high-contrast;
+}
+
+html[data-color-scheme="dark-high-contrast"], html:not([data-color-scheme]) {
+  --color-neutral05: #000;
+  --color-neutral10: #000;
+  --color-neutral15: #000;
+  --color-neutral20: #fff;
+  --color-neutral25: #fff;
+  --color-neutral30: #fff;
+  --color-neutral40: #fff;
+  --color-neutral50: #fff;
+  --color-neutral60: #fff;
+  --color-neutral70: #fff;
+  --color-neutral80: #fff;
+  --color-neutral90: #fff;
+
+  --color-danger0: #ED1741;
+  --color-danger1: #ED1741;
+  --color-danger2: #ED1741;
+  --color-danger4: #ED1741;
+  --color-danger5: #ED1741;
+
+  --color-accent8: #a6ffea;
+  --color-accent7: #a6ffea;
+  --color-accent6: #a6ffea;
+  --color-accent5: #a6ffea;
+  --color-accent4: #a6ffea;
+
+  --color-focus: #a6ffea;
+  color-scheme: dark-high-contrast;
```

Again, same as above.

```suggestion
  color-scheme: dark;
```